features/admin/view_services.feature

```diff
@@ -0,0 +1,26 @@
+@view-a-service
+Feature: View a service
+
+Scenario Outline: Correct user can access the service page
+  Given I am logged in as the production <role> user
+  # TODO GET ME HERE FROM THE LOGIN
+  And I am on the /admin/services/1123456789012346 page
+  Then I am on the '1123456789012346 DM Functional Test N3 Secure Remote Access' page
+
+  Examples:
+    | role               |
+    | admin              |
+    | admin-ccs-category |
+
+Scenario Outline: Correct users cannot access the service page
+  Given I am logged in as the production <role> user
+  # TODO GET ME HERE FROM THE LOGIN
+  And I am on the /admin/services/1123456789012346 page
```

Comment:
In this case I suppose we don't need a real service ID as users should just get blocked before it even hits the API for a service.  So the one in the scenario here could be obviously fake maybe?